proposals/templates/proposals/vue_templates/proposal_list.html

```diff
@@ -73,15 +75,17 @@ <h4>
                      title="{% trans 'Inzien' %}">
             </a>
             <a
-                :href="$url('proposals:archive_status', [proposal.pk])"
-                v-if="proposal.in_archive && context.is_secretary"
+                v-if="!proposal.is_pre_assessment &&
+                proposal.status == context.proposal.DECISION_MADE &&
+                !proposal.status_review &&
+                proposal.children.length === 0"
+                :href="$url('proposals:copy_revision')"
             >
-                <img src="{{ img_hide }}"
-                     title="{% trans 'Verberg' %}">
+                <img src="{{ img_revise }}"
+                     title="{% trans 'Maak revisie' %}">
             </a>
         </template>
```

Comment:
Why are you removing an action for this?

Secondly, you could also implement this by creating a property-method (`is_revisable`) on `Proposal`, and then passing that as a field in the serializer. Honestly, it's not a bad idea to have that property for other checks in Python as well. Just a suggestion, this works fine :)

Reply:
Removing the action was a little mistake. I did like your suggestion of creating an `is_revisable` property method, so that exists now.